deno.json

```diff
@@ -26,5 +26,8 @@
         "wasi/testdata"
       ]
     }
+  },
+  "compilerOptions": {
+    "noImplicitOverride": true
```

Comment:
This change should be added to [strict-ts44.tsconfig.json](https://github.com/denoland/deno_std/blob/main/strict-ts44.tsconfig.json) instead

Reply:
Oh alright, fixed.